CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to Atlas
+
+:tada: To begin with, thank you so much in your interest in contributing to Atlas :tada:
+
+### Contributing TL;DR
+
+1. Always make sure to create an issue/comment on an existing issue before you start any work.
+2. Make sure to follow [Atlas styleguide](docs/styleguide.md)
+3. Once you have a change ready, push to a branch on your own fork
+4. Open a PR from your fork to the `dev` branch on the main repo
+5. Make sure to link to a relevant issue when opening a PR
+6. Have fun!
+
+### Looking for work
+
+If you would like to contribute but don't know where to start, take a look at issues with a [good first issue label](https://github.com/Joystream/atlas/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) or [bug label](https://github.com/Joystream/atlas/issues?q=is%3Aissue+is%3Aopen+label%3Abug). Those are usually quite easy to get into. Also, you are always free to suggest any changes that you think would be beneficial to the app.
+
+### Contributing resources
+
+- [Atlas styleguide](docs/styleguide.md)
+- [Atlas technical overview](docs/overview.md)
+
+### Contributing guidelines
+
+If you want to contribute, the very first place to stop at is the [issues list](https://github.com/Joystream/atlas/issues). It's a very good practice to base any work on an issue. If you want to add some functionality to the app, create an issue with your proposal first, so we can discuss. If it turns out the team disagrees with the change you're proposing, you will not waste time doing wasted work. If you would like to work on an existing issues, you're free to do so, but please first leave a comment in the issue that you will be working on that. This ensures that we don't get into a situation when more than one contributor works on a given feature at the same time.
+
+Once you know what you will be working on and created/commented on an appropriate issue, it's time to get to work. Before you begin tough, please fork the main repo, so you have your own copy which you can use for development. We work in a forking model, so nobody pushes to the main repo directly - all contributors have their own forks from which they open PRs to the main repo.
+
```

Comment:
Not sure, but maybe we could add here a link to this video guide for newcomers? https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github